projects/plugins/super-cache/inc/delete-cache-button.php

```diff
@@ -86,6 +86,17 @@ function wpsc_admin_bar_delete_cache_ajax() {
 		if ( defined( 'WPSCDELETEERROR' ) ) {
 			return json_decode( constant( 'WPSCDELETEERROR' ) );
 		} else {
+			$req_path = !empty($_POST['path']) ? $_POST['path'] : '/';
```

Comment:
Can you add spaces in the empty() call here, so it follows the WordPress coding conventions?  It should look like this:
`! empty( $_POST['path'] )`

It's probably worth sanitizing as well, since that's what happens when `wpsc_after_delete_cache_admin_bar` is used further down in the code.  the default path is '' in the other action, so we should probably change that too. 
The line will look like this:
`$req_path = ! empty( $_POST['path'] ) ? sanitize_text_field( stripslashes( $_POST['path'] ) ) : '';`

Reply:
I just copied full line from code below, to keep consistency.